bisrat_answer/reverseStringAnswer.metta

```diff
@@ -0,0 +1,21 @@
+; If the list is empty, return the accumulated reversed list  
+(= (reverseAcc () $reversed) $reversed)
+
+; else
+(= (reverseAcc $list $reversed)
+       (let* (($first-item (car-atom $list))   ; Extract the first element (head)
+              ($remaining-items (cdr-atom $list)))  ; Get the rest of the list (tail)
+             
+       (reverseAcc $remaining-items (cons-atom $first-item $reversed))
+       )
+)
+
+; helper function
+(= (reverseList $list)
+   ; Start the reversal process with an empty list as the accumulator
+   (reverseAcc $list ())
+)
+
+; Example call:
+! (reverseList (h e l l o))
+! (reverseList(b i s r a t ))
```

Comment:
this function is returning error.

Reply:
on the playground it  is returning the right answer. please re run the code to the result

![image](https://github.com/user-attachments/assets/594d9080-d7a4-4a49-bd55-eb53b9fe1a35)